lib/hcm2/hcm2.js

```diff
@@ -9,7 +9,7 @@ const pDelay = require('delay')
 const BN = require('../bn')
 
 const testNotes = 'false'
-const LDN = 'HCM2_01'
+let LDN = 'HCM2_01'
```

Comment:
We can lowercase this if it's not a const anymore

Reply:
I'd have to change all other lines using it though:

```js
{
  LDN: ldn
}
```

The only reason it was uppercase to begin with.